application/templates/namespace.yaml

```diff
@@ -0,0 +1,9 @@
+{{- if .Values.deployment }}
+{{- if .Values.namespace.enabled -}}
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ .Values.namespace.name }}
+{{- end -}}
```

Comment:
@hanzala1234 add support to specify labels, annotations, etc.

Reply:
Done